### ISSUE TITLE
[feature] option to reload configuration file of the existing redshift process

### ISSUE THREAD

**Issue by dqgls** [jonls/redshift#412]
In some situations you might want to change the configuration, for example when you're in more than usual darkness or have a headache. Let's say you already have an alternative configuration file with lower temperatures and lower gamma just for this purpose, so you simply copy it to `~/.config/redshift.conf` (default config location). So far so good.

Now, the problem is that there's no way to make the already running redshift process reload its configuration, and smoothly transition to it (assuming `transition=1`). You have to kill the existing process and start redshift again. This momentarily glares the default white-blue screen light at your face, which is quite annoying for the eyes in such a situation.

---

So basically what would need to be implemented is a way to communicate to an already running redshift process, and tell it reload variables from `~/.config/redshift.conf`. Something like `redshift --reload`.

This could be done for example by sending a specific unix signal (see `kill -l` for a list) to the existing redshift process, which then calls its config init/reload function. I see the code already has some signal handling, so it might not be that hard to add this feature?

**Comment by jonls** [comment]
Thanks for the suggestion. I think this was already proposed in #96. Please continue the discussion there.
